src/main/resources/db/migration/V3_28__add_actioned_by_prison_exclude_date.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE prison_exclude_date ADD actioned_by VARCHAR(60);
```

Comment:
Is 60 enough, whats the limit on usernames?

Reply:
usernames are usually 8. kept it 60 as username on actioned_by table also has user_name as 60.